packages/cycle-scripts-one-fits-all/scripts/start.js

```diff
@@ -0,0 +1,14 @@
+'use strict'
+
+const webpack = require('webpack')
+const WebpackDevServer = require('webpack-dev-server')
+
+const port = 8080
+process.env.NODE_ENV = 'development'
+
+const config = require('./configs/webpack.config.js')
```

Comment:
I've tried to run it and it throws the following errors:

```
fallbackLoader option has been deprecated - replace with "fallback"
loader option has been deprecated - replace with "use"
fallbackLoader option has been deprecated - replace with "fallback"
loader option has been deprecated - replace with "use"

testApp/node_modules/webpack-merge/lib/join-arrays-smart.js:124
  var _entry$loader$match = entry.loader.match(loaderNameRe),
```

Some how your setup is generate a webpack1 configuration while you running on webpack2